src/app/layout/NavigationDrawer/NavigationDrawer.js

```diff
@@ -15,6 +15,7 @@ import { Link, useHistory } from "react-router-dom";
 import { useFirebase } from "react-redux-firebase";
 
 import { PrivateComponent } from "../../component";
+import { ListItemIcon } from "@material-ui/core";
 
 export default function TemporaryDrawer() {
   const firebase = useFirebase();
```

Comment:
I think this firebase is not used right now, maybe we can remove some unused import

Reply:
Sure. I can do that! ☺️

Reply:
@mat10tng I have updated styles to useStyles and there is a logout feature which is being used by firebase for signout so I am unable to remove it.